src/main/java/hw14/ArrayImplementer.java

```diff
@@ -0,0 +1,82 @@
+package hw14;
+
+import java.util.*;
+
+public class ArrayImplementer {
+    public static List createList() {
+        List<String> words = new ArrayList<>();
+        words.add("apple");
+        words.add("banana");
+        words.add("cherry");
+        words.add("apple");
+        words.add("banana");
+        words.add("cherry");
+        words.add("apple");
+        words.add("banana");
+        words.add("cherry");
+        words.add("date");
+        words.add("elderberry");
+        return words;
+    }
+
+    public static int countOccurrence(List<String> words, String word) {
+        int count = 0;
+        for (String str : words) {
+            if (str.equals(word)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static void calcOccurrence(List<String> words) {
+        Map<String, Integer> occurrence = new HashMap<>();
+        for (String word : words) {
+            if (occurrence.containsKey(word)) {
+                occurrence.put(word, occurrence.get(word) + 1);
+            } else {
+                occurrence.put(word, 1);
+            }
+        }
+        for (Map.Entry<String, Integer> entry : occurrence.entrySet()) {
+            System.out.println(entry.getKey() + " - " + entry.getValue());
+        }
```

Comment:
get rid of this, please.Do all print statements in the `main` method. Make this method return a map instead.